e2e/generateIndex.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+echo "Generating index.ts file..."
+echo "import * as inversifyConfig from './inversify.config';
+export { inversifyConfig };
+export * from './inversify.types';
+export * from './TestConstants';
+" > index.ts
+
+listOfDirs="driver utils pageobjects"
+listOfExcludes="./driver/CheReporter.ts"
+for dir in $listOfDirs
```

Comment:
Does this script work with nested folders? For example "pageobjects" -> "dashboard"

Reply:
In short yes - there is already index.ts file generated, you can take a look https://github.com/eclipse/che/pull/14281/files#diff-14f7a35f8fc13dbe57f6822026fa0f1dR12